values.yaml

```
@@ -77,6 +77,70 @@ global:
   # Enable security context constraints required for OpenShift
   sccEnabled: false
 
+  # Deploy auth sidecar to use openshift native features
+  authSidecar:
+    enabled: false
+    image: nginxinc/nginx-unprivileged
+    pullPolicy: IfNotPresent
+    default_nginx_settings: |
```

I think these defaults should be in the template, not here. The config here should be.
```suggestion
    nginx_settings: ~
```
Then the template should have `{{- if .Values.global.authSidecar.nginx_settings }}`